### DISCUSSION TITLE
Inability to get into the armory for other crew members.

### DISCUSSION THREAD

**Discussion by Anamikinaru** [FakeFishGames/Barotrauma#8951] [Ideas and suggestions]
Please make it possible at least in the campaign to allow characters to enter certain rooms.

**Comment by Oukido** [comment]
There are 2 ways you can help yourself in singleplayer that do not involve any console commands:

You can switch ID cards with the captain to give another "main" character access to the armory. There seems to be a bug that the captain regains access to the armory even with the other id card equpped. (not sure if it is just finishing a round or maybe save+quit and reload)

Install a circuit to control the door. Can be a simple switch or maybe a motion detector.
(if it is multiplayer and you trust your shipmates you can enable wiring in the server settings)

**Reply by Anamikinaru** [reply]
Thanks for the answer.
